### DISCUSSION TITLE
定义和定理格式问题

### DISCUSSION THREAD

**Discussion by pkeke** [note286/xduts#105] [Q&A] [answered]
请问一下像下面这样定义中引用加小括号有问题嘛，还有定义的前后间距，没找到相关的文件说明。
![7c72fbf4a3344e57d0e43a71700dec6](https://user-images.githubusercontent.com/44956389/227957027-99b6928c-dcaa-47bd-9fbe-200b4f741e46.png)

**Comment by note286** [comment, marked as answer]
首先，你给出的排版效果是否合理请咨询你的导师，我无法解答。其次，我已经在 https://github.com/note286/xduts/discussions/87 中回答你了啊；关于定理的设置，我也在 https://github.com/note286/xduts/discussions/40#discussioncomment-4982134 给出了一些样例。

```latex
\documentclass{xdupgthesis}
\xdusetup{info/bib-resource=reference.bib}
\usepackage{amsthm}
\newtheoremstyle{xdu}{3pt}{3pt}{}{}{\bfseries}{}{.5em}{\qquad\thmname{#1}\thmnumber{ #2}\thmnote{ #3}}
\theoremstyle{xdu}
\newtheorem{definition}{定义}[chapter]
\begin{document}
\chapter{title}
测试测试测试测试测试测试测试测试测试测试测试测试测试测试
测试测试测试测试测试测试测试测试测试测试测试测试测试测试
\begin{definition}[\parencite{key}]
测试测试测试测试测试测试测试测试测试测试测试测试测试测试
测试测试测试测试测试测试测试测试测试测试测试测试测试测试
\end{definition}
测试测试测试测试测试测试测试测试测试测试测试测试测试测试
测试测试测试测试测试测试测试测试测试测试测试测试测试测试
\end{document}
```

![image](https://user-images.githubusercontent.com/16207292/227961637-f0d2dc3f-10a3-4272-a287-4846a9a8f624.png)

**Reply by pkeke** [reply]
谢谢！
